docs/docs/tutorial/part-4/index.mdx

```diff
@@ -462,7 +461,6 @@ const Layout = ({ pageTitle, children }) => {
 
   return (
     <div className={container}>
-      <title>{pageTitle} | {data.site.siteMetadata.title}</title>
```

Comment:
Moving this to Head API means we lose access to `siteMetaData` 

To fix that, we'd need to introduce additional code to get `siteMetaData` ( useStaticQuery / Page Query)

Another thing we can do is to completely remove " | `{data.site.siteMetadata.title}`"

What would you suggest we do here

Reply:
Let's remove it from the `<title>` tag then. Otherwise it would complicate things too much